pkg/util/merr/errors.go

```diff
@@ -44,10 +44,13 @@ var (
 	ErrServiceForceDeny            = newMilvusError("force deny", 9, false)
 
 	// Collection related
-	ErrCollectionNotFound         = newMilvusError("collection not found", 100, false)
-	ErrCollectionNotLoaded        = newMilvusError("collection not loaded", 101, false)
-	ErrCollectionNumLimitExceeded = newMilvusError("exceeded the limit number of collections", 102, false)
-	ErrCollectionNotFullyLoaded   = newMilvusError("collection not fully loaded", 103, true)
+	ErrCollectionNotFound          = newMilvusError("collection not found", 100, false)
+	ErrCollectionNotLoaded         = newMilvusError("collection not loaded", 101, false)
+	ErrCollectionNumLimitExceeded  = newMilvusError("exceeded the limit number of collections", 102, false)
+	ErrCollectionNotFullyLoaded    = newMilvusError("collection not fully loaded", 103, true)
+	ErrAliasNotFound               = newMilvusError("alias not found", 104, false)
```

Comment:
If they're "collection related" errors, please name these following `ErrCollectionXxx` format, to make it easier to find them for developer